packages/xo-server-sdn-controller/src/index.js

```diff
@@ -304,44 +304,44 @@ class SDNController extends EventEmitter {
     }
   }
 
-  async _objectsAdded(objects) {
-    await Promise.all(
-      map(objects, async object => {
-        const { $type } = object
+  _objectsAdded(objects) {
+    map(objects, object => {
```

Comment:
`forEach`

Reply:
`objects` is not iterable so I can't use `_.forEach`.
Should I use `lodash#forOwn`?

Reply:
How does `_.map` manage to iterate over it if it's not iterable?

Reply:
It's `lodash#map`

Reply:
So you can use lodash's `forEach`, right?

Reply:
Oh yes, I didn't know it existed ^^'.